### PR TITLE
Added option for a4paper size and adjust vspace accordingly

### PR DESCRIPTION
**Pull request by fergul**
Thanks for latexing this template. I needed a4 size paper so I fixed issue #1 .

### CODE REVIEW

File: mcdowellcv.cls [hunk 1]
```diff
@@ -29,30 +29,44 @@
   \def\mainfontface{Calibri}
 }
 
+% Set up papersize according to options, default to US letter
+\def\papersize{letterpaper}
+%Set up bool for adjusting vspacing later
+\newif\ifpapersize
```
Comment (dnl-blkv) [inline mcdowellcv.cls]: There are many paper formats than `letterpaper` and `a4paper`. In order to allow for those, we can use some package supporting the `switch` construct, such as `boolexpr`, and then just switch over the `\papersize` wherever needed. (source: http://tex.stackexchange.com/questions/29133/how-to-create-switch-structure-comparing-strings-in-latex)

**Comment by dnl-blkv** [comment]
@fergul Should've started with it, but better late than never: thanks for the contribution! It is very much appreciated.

**Comment by fergul** [comment]
@dnl-blkv No problem, I absolutely agree. I didn't have much time and took my path of least resistance - thinking those were the only real two paper size choices :P. Unfortunately I don't think I will have much time in the near future to act upon your suggestions.

**Comment by otiofrui** [comment]
@fergul @dnl-blkv is there any chance this is getting fixed? Would be greatly appreciated, if A4 would work.

**Review by dnl-blkv — APPROVED** [review]
conflicts resolved

**Comment by dnl-blkv** [comment]
@fergul @goseind merged :)

**Comment by dnl-blkv** [comment]
Hope it works xD
Don't have a LaTeX setup to test it right now...

**Comment by goseind** [comment]
Sorry for the very late reply, I've now tested it and it works! Thanks very much 😄

**Pull request merged**